be/src/exec/vectorized/schema_scan_node.cpp

```diff
@@ -122,6 +124,7 @@ Status SchemaScanNode::prepare(RuntimeState* state) {
                          << dest_slot_descs[i]->type();
             return Status::InternalError("schema not match.");
         }
+        _index_map[i] = j;
```

Comment:
Why src and dest slots could unmatch?

Reply:
in current brach fe slot id generator starts from 1,
in version <= 1.18.2 fe slot id generator starts from 0.